### DISCUSSION TITLE
Help, help Hubs Cloud

### DISCUSSION THREAD

**Discussion by soporteviro360** [mozilla/hubs#5087] [Hubs Cloud Questions]
Friends, I already have cloud hubs, I already clone the repository on my localhost, I already have VSCode linked to the cloned repository.
The changes are uploaded to the repository.
But Hubs doesn't show the changes.
For example I change the Title but it doesn't take the changes.
Thanks in advance for your comments
almonato

**Comment by pattersonbl2** [comment]
Could you give us a screenshot of the issue?   I am not sure if this is a git issue or a code issue.

**Reply by soporteviro360** [reply]
Hello Pattersonbl2,
My situation is the next:
git clone --> OK
git remote add upstream https://github.com/mozilla/hubs.git --> OK
upstream https://github.com/mozilla/hubs.git (fetch)
upstream https://github.com/mozilla/hubs.git (push)
-->OK
Run your custom client
npm ci --> OK
npm run login --> OK
npm run deploy --> ERROR - Freezes

I don't know if changing the index and placing it in Spanish is the way I'm doing.

How can I modify the index?

Thanks

Alfredo Villa

**Reply by soporteviro360** [reply]
My situation is the next:
git clone --> OK
git remote add upstream https://github.com/mozilla/hubs.git --> OK
upstream https://github.com/mozilla/hubs.git (fetch)
upstream https://github.com/mozilla/hubs.git (push)
-->OK
Run your custom client
npm ci --> OK
npm run login --> OK
npm run deploy --> ERROR - Freezes

I don't know if changing the index and placing it in Spanish is the way I'm doing.

How can I modify the index?

Thanks

**Reply by soporteviro360** [reply]
Help Please

**Reply by pattersonbl2** [reply]
It's hard to figure out what the issues could be. We don't run custom clients by nature. a reason could be related to the front end not connecting to the back-end code. This question would be great to also ask in our community discord. You can join here (https://discord.com/invite/dFJncWwHun). I will ask my team if they may have a quick answer as well.
